insights/specs/default.py

```diff
@@ -375,7 +375,7 @@ def httpd_cmd(broker):
     ls_disk = simple_command("/bin/ls -lanR /dev/disk")
     etc_and_sub_dirs = sorted(["/etc", "/etc/pki/tls/private", "/etc/pki/tls/certs",
         "/etc/pki/ovirt-vmconsole", "/etc/nova/migration", "/etc/sysconfig",
-        "/etc/cloud/cloud.cfg.d"])
+        "/etc/cloud/cloud.cfg.d", "/etc/rc.d/init.d"])
     ls_etc = simple_command("ls -lan {0}".format(' '.join(etc_and_sub_dirs)))
```

Comment:
@vishwanathjadhav I realize this is not part of your changes, but since it is related, would you mind to change this command from `ls` to `/bin/ls`?

Reply:
@bfahr - Done the changes